src/transforms/coerce.jl

```diff
@@ -0,0 +1,34 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    Coerce(pairs, tight=false, verbosity=1)
+
+Return a copy of the table, ensuring that the element scitypes of the columns match the new specification.
+
+Valid specifications (more to be added):
+
+(1) one or more column_name=>Scitype pairs
```

Comment:
We just need to mention the Scitypes.coerce function in the docstring, perhaps give examples of how to create the Coerce transform instead of a generic spec.